### DISCUSSION TITLE
2006/02/25/%E5%BC%80%E6%94%BE%E5%AD%98%E5%8F%96

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#76] [Announcements]
# 2006/02/25/%E5%BC%80%E6%94%BE%E5%AD%98%E5%8F%96

有一件让我觉得不爽的事就是发表论文还要出钱。这发表论文是把自己的知识分享给别人，是在做贡献，没有收入就算了，还要掏钱。现在，去发表文找的人恐怕就没多少是以分享知识给别人为根本目的了。

https://blog.qizhen.xyz/2006/02/25/%E5%BC%80%E6%94%BE%E5%AD%98%E5%8F%96

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Yan
2006年03月1日 18:58 编辑
愤青啊~~
